model/riscv_insts_fext.sail

```diff
@@ -368,45 +336,33 @@ mapping clause encdec = STORE_FP(imm7 @ imm5, rs2, rs1, DOUBLE)
 
 /* Execution semantics ================================ */
 
-val process_fstore : (xlenbits, MemoryOpResult(bool)) -> Retired
-function process_fstore(vaddr, value) =
-  match value {
-    MemValue(true)  => { RETIRE_SUCCESS },
-    MemValue(false) => { internal_error(__FILE__, __LINE__, "store got false from mem_write_value") },
-    MemException(e) => { handle_mem_exception(vaddr, e); RETIRE_FAIL }
-  }
-
 function clause execute (STORE_FP(imm, rs2, rs1, width)) = {
+  let width_bytes = size_bytes(width);
+
+  // This is checked during decoding.
+  assert(width_bytes <= sizeof(xlen_bytes)); // TODO: Should this actually be flen?
```

Comment:
Yes? That's what was being checked in the code you deleted

Reply:
Ooops, slight Git mistake - I didn't actually intend to include the fext changes since they don't quite work yet. It only compiles because of the mistake on line 302 -  `2` should be `width_bytes`. With that change it can't compile because `nan_box` isn't generic (needs #448).

I removed these changes.